stdlib/public/Python/Python.swift

```diff
@@ -1253,6 +1253,21 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject : IteratorProtocol, Sequence {
+
```

Comment:
Remove extra line.

Reply:
This is not valid anymore right?

Reply:
This is no longer valid once the empty line is used to separate a property declaration and a function.